src/T2DMap.cpp

```diff
@@ -688,7 +688,7 @@ void T2DMap::initiateSpeeWalk(const int speedWalkStartRoomId, const int speedWal
         if (mpMap->findPath(speedWalkStartRoomId, speedWalkTargetRoomId)) {
             mpHost->startSpeedWalk();
         } else {
-            mpHost->mpConsole->printSystemMessage(QStringLiteral("%1\n").arg(tr("Mapper: Cannot find a path from %1 to %2 using known exits.\n")
+            mpHost->mpConsole->printSystemMessage(QStringLiteral("%1\n").arg(tr("Mapper: Cannot find a path from %1 to %2 using known exits.")
```

Comment:
The effect of the original code was to leave a blank line *after* the message.  To achieve the same effect but remove the need for it from the translation we should, perhaps, put a *second* `\n` in the text for the `QStringLiteral(`...`)` . Any opinions from any @Mudlet/ui-review team members?

Reply:
For this early PR from a new contributor, better to stay on focus of the original ticket. I don't think we need more blank lines though, aesthetically speaking.

Reply:
There were one (1) new line earlier, there will be one new line in the future, only by accident two (2) new lines were put here. This will fix that again.

Reply:
There was a newline at the end of the non-translated text that is needed to ensure that the following text is started on a new line - but **also** a *second* new line at the end of the translated text which would ensure that there is a blank or empty line before the following text.  To reproduce that but not require the translator to provide it you would need to add a **second** new-line in the non-translated text.  This is, of course, unless the intent of the PR is to **remove** the blank line that would follow the `"Mapper: Cannot find a path from %1 to %2 using known exits."` message?

Reply:
The second newline was only added by accident in #3503